### PR TITLE
Change auto update to false

### PR DESCRIPTION
**Pull request by evaldofelipe**
This PR solve a issue created at PR #7 when the default value was not changed.

### CODE REVIEW

File: files/etc/apt/apt.conf.d/10periodic [hunk 1]
```diff
@@ -1,4 +1,4 @@
-APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Update-Package-Lists "0";
 APT::Periodic::Download-Upgradeable-Packages "0";
 APT::Periodic::AutocleanInterval "0";
 APT::Periodic::Unattended-Upgrade "0";
```
Comment (lborguetti) [inline files/etc/apt/apt.conf.d/10periodic:4]: Can you add new line here?

Thanks

**Review by lborguetti — CHANGES_REQUESTED** [review]
LGMT

File: files/etc/apt/apt.conf.d/10periodic (hunk 1 above)
Comment (evaldofelipe) [reply files/etc/apt/apt.conf.d/10periodic:4]: This file as default don't have a new line.

This new line are really needed?

Comment (lborguetti) [reply files/etc/apt/apt.conf.d/10periodic:4]: No :+1:

**Review by lborguetti — APPROVED** [review]

**Pull request merged**